drunc_docker_service/Dockerfile

```diff
@@ -12,13 +12,12 @@ WORKDIR /basedir
 # See https://github.com/DUNE-DAQ/drunc/wiki/Setup-drunc-with-DUNE-DAQ for explanation
 RUN source /cvmfs/dunedaq.opensciencegrid.org/setup_dunedaq.sh && \
         setup_dbt latest_v5 && \
-        dbt-create fddaq-v5.1.0-a9 fddaq-v5.1.0-a9 && \
-        cd fddaq-v5.1.0-a9 && \
+        dbt-create -n NFD_DEV_241114_A9 && \
+        cd NFD_DEV_241114_A9 && \
```

Comment:
Seems like this PR was made before the change in #225 
We will need to add the equivalent line from that PR now that this path has changed

Reply:
Yep. That branch was up to date with `main`, incorporated the change in #225